ExceptionTryCatchFinally.Program..ctor[instance.void..()]
Trace:
	[start] ExceptionTryCatchFinally.Program..ctor[instance.void..()]
	[ end ] ExceptionTryCatchFinally.Program..ctor[instance.void..()]


ExceptionTryCatchFinally.Program.<Main>g__ThrowException|0_0[void..()]
Trace:
	[start] ExceptionTryCatchFinally.Program.<Main>g__ThrowException|0_0[void..()]
		[start] ExceptionTryCatchFinally.Program..ctor[instance.void..()]
		[ end ] ExceptionTryCatchFinally.Program..ctor[instance.void..()]
	[ end ] ExceptionTryCatchFinally.Program.Main[void..(class.System.String[])]


